pml/element.py

```diff
@@ -23,32 +28,106 @@ def __init__(self, name, length, element_type, physics=None):
         self._devices = dict()
         self._enabled = True
 
+    def __repr__(self):
+        """Auxiliary function to print out an element.
+
+        Prints out the set of families an element is part of, as a string.
+
+        Returns:
+            string: Set of families the element is part of.
+        """
+        return str(self.families)
+
     def get_fields(self):
+        """Get the required fields to access the devices of an element.
```

Comment:
I would say: 'Get the fields defined on the element.'
And: 'A sequence of all the fields defined on the element.'